### PR TITLE
Add aarch64 support

### PR DESCRIPTION
**Pull request by nerohmot**
Checklist
* [X] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [X] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by nerohmot** [comment]
@conda-forge-admin, please rerender

File: conda-forge.yml [hunk 1]
```diff
@@ -7,3 +7,5 @@ travis:
   secure: {BINSTAR_TOKEN: MJeUCT1LOGBa6H6rLCHZiClpZxdejtcgne/LRpcqdL/gO79DTSC8PGVOKuuhi7vq3kU4CUaDGZviTwUyNc1LrMGMBpYeA32MGVsrlBOm3mp6bmSY06AawOowtEn9qlj9BA/XAlVTHKri1VjmVbiJDmn35s4koLkbaiZaIw5XmbPx2UG174G/PjliD+Sn4e6ynSR03a71qoYij+kVeJT6s/dwZGbePpBEEtd8su/f2dZRtkrMsPB8rh+Tzj3DXojv8BgUkeNCIWgpu2qPITeoSQkDH2YKREtiSdD1g12i11y3amAlhZFRC9gZEekPw8HD7jjJmbg9czjZlJTbVm+trJbnAuI+EgRfMgQ6T57UP22SSUoGbSeZgRezSBP1JF5rJMno2jd1pxIshyQ93oRx7WYf1Ev9B8Pa5py9pE/cIYkGLrCZbX6VIyUVBXiGCXtfTrBuou5Z4LvNMn0xnYjk7xRLFPLXaQQ04bwvP+sM7vA7y6LPVDX8LxCXpL1tYlK+ctyIIKx4TVgqcay2UCqRMS2shoHU11CdAdbrVvmVEZjXgG0JEUMRUSxPdrZraoREDdiHUDxB2hmcbis3wItKBNnHtYEEnNlj7gJzq/xfMJqylsDqWbtD9VJA7BgSFdkQ44Oi1lOnhRwwfa7GRxWU/o8qjpVpVNuv65DNxJGVEDM=}
 provider:
   win: azure
+  linux_aarch64: azure
```
Comment (ccordoba12) [inline conda-forge.yml]: I think we should leave the default provider, as was suggested in the Wurlitzer feedstock.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -12,7 +12,7 @@ source:
   {{ hash_type }}: {{ hash }}
 
 build:
-  number: 0
+  number: 1
```
Comment (ccordoba12) [inline recipe/meta.yaml]: Please don't increase the build numbers, it's not really necessary.

Comment (nerohmot) [reply recipe/meta.yaml]: Yes, I know now.
Shall I do a new PR ?

Comment (ccordoba12) [reply recipe/meta.yaml]: Don't worry, I can do these changes myself.

**Comment by ccordoba12** [comment]
@conda-forge-admin, please rerender

**Comment by ccordoba12** [comment]
@nerohmot, you need to add aarch64 support for the `pylint` and `ujson` packages for this one to have support for that architecture.

**Comment by ccordoba12** [comment]
This was done in PR #46.

**Pull request closed without merging**